extensions/Worker.Extensions.CosmosDB/CosmosDBEventTriggerAttribute.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.CosmosDB
+{
+    public sealed class CosmosDBEventTriggerAttribute : TriggerBindingAttribute
```

Comment:
nit: should the name be `CosmosDbEventTriggerAttribute`? 
Similar to [`DbException`]( https://docs.microsoft.com/en-us/dotnet/api/system.data.common.dbexception.-ctor?view=net-5.0) or [`SqlDbType`](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqldbtype?view=net-5.0).

Reply:
That's a good point! Though, I've been trying to keep it consistent with the types we have in `WebJobs` that are used today by .NET Core 3 functions. Example - https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.CosmosDB/CosmosDBAttribute.cs